Move aliases to general config (#9879)
deskflow/deskflow: docs/user/configuration.md

 - You can use both the ''switchDelay'' and ''switchDoubleTap'' options at the same time. Deskflow will switch when either requirement is satisfied.

### Screen Settings

Each screen will have a section where its configuration will be stored, if the screen was named "foo" the section will be named `[foo]`

|Option              |    Valid Values    |Description|
|:-------------------|:------------------:|:-----------|

> | aliases            | comma seperated list of host names | names here will be used as alternate names for the screen, Names must compliant hostnames.|

Reviewer: ```suggestion
| aliases            | Comma separated list of hostnames | Names here will be used as alternatives for the computer. Names must be valid hostnames. |
```

Comments are introduced by ''#'' and continue to the end of the line. ''name'' must be one of the following:

* ''screens''
* ''links''
* ''options''

> The file is parsed top to bottom and names cannot be used before they've been defined in the <code>screens</code> or As an alias in the general config. So the <code>links</code> must appear after the <code>screens</code>.

Reviewer: ```suggestion
The file is parsed top to bottom and names cannot be used before they've been defined in the <code>screens</code> or as an alias in the general config. So the <code>links</code> must appear after the <code>screens</code>.
```

### Example textual configuration file

> the alias section is no longer in the server config

Reviewer: ```suggestion
The alias section is no longer in the server config.
```
